kommander: Update supported k8s versions chart for platform services (#3393)
mesosphere/dcos-docs-site: pages/dkp/kommander/1.3/workspaces/workspace-platform-services/index.md

excerpt: How workspace platform services work
---

When attaching a cluster, Kommander federates certain platform services on the newly attached cluster. Operators can use the Kommander UI to customize which platform services to deploy to the attached clusters in a given workspace.

On attachment, three factors impact successfully deploying a platform service on the attached cluster:

1. Is the attached cluster a Konvoy cluster or not? For example, a cluster deployed using AWS EKS.
2. Is the platform service enabled in the Workspace Platform Services settings?
3. Does the platform service support the version of Kubernetes running on the cluster?

> The following tables describe the list of platform services that are federated on attachment, along with the versions of Kubernetes that they support. If the cluster is running a version of Kubernetes that is not in the supported list, then the platform service will not be federated to that cluster. If the platform service description indicates `only federated on non-Konvoy clusters`, the platform service will not be installed into Konvoy clusters, since it is by default installed with Konvoy via [Kubernetes Base Addons](/dkp/konvoy/1.7/addons/).

Reviewer: ```suggestion
The following tables describe the list of platform services that are federated on attachment, along with the versions of Kubernetes that they support. If the cluster is running a version of Kubernetes that is not in the supported list, the platform service is not federated to that cluster. If the platform service description indicates `only federated on non-Konvoy clusters`, the platform service is not installed into Konvoy clusters, since it is by default installed with Konvoy using [Kubernetes Base Addons](/dkp/konvoy/1.7/addons/).
```

Currently, the monitoring stack is federated by default and the logging stack is not.

Review the [workspace platform service resource requirements](/dkp/kommander/1.3/workspaces/platform-service-requirements/) to ensure that the attached clusters have sufficient resources.

### Customize a workspace's platform services

> Provided you have access to edit a workspace, you can customize the platform services that are federated to a workspace's clusters using the Kommander UI. You can access the settings page by going to the desired workspace's **Dashboard** page, locating the Workspace card, clicking the **Actions** drop-down button and selecting **Edit Workspace Platform Services**. You can also go to the Workspace Dashboard's **Actions** menu button to access the settings page.

Reviewer: ```suggestion
If you have access to edit a workspace, you can customize the platform services that are federated to a workspace's clusters using the Kommander UI. Access the settings page by going to the specific workspace's **Dashboard** page, locating the Workspace card, selecting the **Actions** drop-down button and selecting **Edit Workspace Platform Services**. You can also go to the Workspace Dashboard's **Actions** menu button to access the settings page.
```

excerpt: How workspace platform services work
---

> ## Workspace platform services

Author: maybe remove since redundant?
```suggestion
```
